src/OpenTelemetry/Trace/Export/Internal/BroadcastActivityProcessor.cs

```diff
@@ -0,0 +1,105 @@
+﻿// <copyright file="BroadcastActivityProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Trace.Export.Internal
+{
+    internal class BroadcastActivityProcessor : ActivityProcessor, IDisposable
```

Comment:
The name is misleading since "broadcast" normally means there is no need to specify the target.

Probably borrow from Java/Python:
* Java [MultiSpanExporter](https://github.com/open-telemetry/opentelemetry-java/blob/bcacf39a9c89cb87e3ce89b19ab047044fa468c6/sdk/src/main/java/io/opentelemetry/sdk/trace/export/MultiSpanExporter.java#L35)
* Python [SynchronousMultiSpanProcessor](https://github.com/open-telemetry/opentelemetry-python/blob/d22ed887e5681f43e411e901cd3eafa78ad19e28/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L104)

Reply:
FanOutActivityProcessor? 🤷